server/card.go

```diff
@@ -0,0 +1,69 @@
+package server
+
+import "github.com/mikitachab/score-board/db"
+
+// PlayerStat represents single stat
+// in Histogram
+type PlayerStat struct {
+	Name  string
+	Value int
+}
+
+// Histogram is used to display stats
+type Histogram struct {
+	Name          string
+	Stats         []PlayerStat
+	TopPlayerName string
+}
+
+// Card is used to represent information
+// for single play on main page
+type Card struct {
+	Histograms []Histogram
+}
+
+func constructCardFromPlayScores(playScores []db.PlayScore) Card {
+	histograms := []Histogram{
+		Histogram{"Military", nil, "Not set"},
+		Histogram{"Wonders", nil, "Not set"},
+		Histogram{"Treasure", nil, "Not set"},
+		Histogram{"Civil", nil, "Not set"},
+		Histogram{"Science", nil, "Not set"},
+		Histogram{"Commerce", nil, "Not set"},
+		Histogram{"Guilds", nil, "Not set"},
+	}
+
+	for _, ps := range playScores {
+		histograms[0].Stats = append(histograms[0].Stats, PlayerStat{ps.Player.Name, ps.SevenWondersScore.MilitaryConflicts})
```

Comment:
looks like we need kind of an iterator through scores (should be done in future development)

Reply:
added issue #34